src/main/java/org/schemaspy/cli/DbmsCommandLineArguments.java

```diff
@@ -0,0 +1,283 @@
+package org.schemaspy.cli;
+
+//import com.beust.jcommander.DynamicParameter;
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.Parameters;
+import org.schemaspy.cli.converters.PatternConverter;
+import org.schemaspy.cli.converters.PropertiesConverter;
+
+import java.util.*;
+import java.util.regex.Pattern;
+
+@Parameters(resourceBundle = "dbmscommandlinearguments")
+public class DbmsCommandLineArguments {
+
+    @Parameter(
+            names = {
+                    "-t", //"--database-type", "database-type",
+                    "schemaspy.t"//, "schemaspy.database-type"
+            },
+            descriptionKey = "databaseType"
+    )
+    private String databaseType;
+    @Parameter(
+            names = {
+                    "-host", //"--host", "host",
+                    "schamspy.host"
```

Comment:
Misspelling

Reply:
Horrible misspelling, good catch, thanks for having a look.